nifi-nar-bundles/nifi-shopify-bundle/nifi-shopify-processors/src/main/java/org/apache/nifi/processors/shopify/GetShopify.java

```diff
@@ -322,6 +325,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                         updateState(session, updatedStateMap);
                     }
                     if (objectCountHolder.get() > 0) {
+                        session.putAttribute(flowFile, CoreAttributes.MIME_TYPE.key(), "application/json");
```

Comment:
Calling `session.putAttribute()` should update the reference to `flowFile`:
```suggestion
                flowFile = session.putAttribute(flowFile, CoreAttributes.MIME_TYPE.key(), "application/json");
```